test/functional/wallet_keypool.py

```diff
@@ -133,6 +133,7 @@ def run_test(self):
 
         nodes[0].walletpassphrase('test', 100)
         nodes[0].keypoolrefill(100)
+        assert_raises_rpc_error(-8, "Invalid parameter, expected valid size.", nodes[0].keypoolrefill, -1)
```

Comment:
@brunoerg I don't think this makes sense as `keypoolrefill` is clearly set to **100** above.

Reply:
Why not? I just calling `keypoolrefill` with an invalid parameter to test how it deals with it, this call doesn't interfere on the previous one (`keypoolrefill(100)`), itkeeps valid and working, that's why the test doesn't fail. Regardless where I put it, it doesn't interfere on the test. Am I missing anything?

Reply:
@brunoerg Then this assert doesn't actually "expects valid size", there are lot of places to put such calls as well, it just doesn't seem necessary for me and that's just my opinion.